src/cue/cue.cc

```diff
@@ -156,13 +167,15 @@ bool CueLoader::load (const char * cue_filename, VFSFile & file, String & title,
                     tuple.set_str (Tuple::Genre, s);
             }
 
-            Rem * rem = cd_get_rem (cd);
+            Rem * rem = track_get_rem (cur);
 
             if (rem)
             {
                 const char * s;
-                if ((s = rem_get (REM_DATE, rem)))
```

Comment:
I guess this was redundant since it's already set in the base_tuple.  Good catch.